google/cloud/bigquery/table.py

```diff
@@ -1011,6 +1011,13 @@ def _build_resource(self, filter_fields):
         """Generate a resource for ``update``."""
         return _helpers._build_resource_from_properties(self, filter_fields)
 
+    def __eq__(self, other):
+        if not isinstance(other, Table):
+            return NotImplemented
+        return self.path == other.path
```

Comment:
This is correct, but involves a lot of computation.

Is there a reason you can't just compare the `tableReference` property?

Reply:
I'm not too concerned about that overhead, and the path seemed just the right property to compare, short and clean. But I'm fine with either way, I'll change.